src/storage-types/src/sources.rs

```diff
@@ -222,8 +228,9 @@ impl<R: ConnectionResolver> IntoInlineConnection<IngestionDescription, R>
 
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
 pub struct SourceExport<S = ()> {
-    /// The index of the exported output stream
-    pub output_index: usize,
+    /// If a subsource, the index in the external publication details that this
```

Comment:
```suggestion
    /// If a subsource, the index in the external table details that this
```
(not all sources use publications)